readme.md

```diff
@@ -33,7 +33,7 @@ conda activate openoa-env
 
 #### Special Note for users of Microsoft Windows:
 
-The Anaconda python distribution is *required* for users of Microsoft Windows. This is because the pip package of GDAL for Windows requires Visual Studio to compile some of the dependencies. While advanced users are welcome to explore this option, we find it is easier to install the following packages via Anaconda:
+If using the 'AssetData' class, the Anaconda python distribution is *required* for users of Microsoft Windows. This is because of the GDAL package used in this class. The pip package of GDAL for Windows requires Visual Studio to compile some of the dependencies. While advanced users are welcome to explore this option, we find it is easier to install the following packages via Anaconda:
```

Comment:
I might change single quotes to using back quotes (the other option on the tilde button) around Asset Data, so it displays in monospace like this: `AssetData`, compared to quoted like this: 'AssetData'. That's more stylistic, but helps to signal that this is a piece of the code.